chart/templates/statefulset.yaml

```diff
@@ -171,9 +171,20 @@ spec:
             timeoutSeconds: {{ .Values.controlPlane.statefulSet.probes.startupProbe.timeoutSeconds }}
             failureThreshold: {{ .Values.controlPlane.statefulSet.probes.startupProbe.failureThreshold }}
           {{- end }}
-          {{- if .Values.controlPlane.statefulSet.security.containerSecurityContext }}
+          {{- if or .Values.controlPlane.statefulSet.security.containerSecurityContext .Values.controlPlane.advanced.kubeVip.enabled }}
           securityContext:
-{{ toYaml .Values.controlPlane.statefulSet.security.containerSecurityContext | indent 12 }}
+            {{- $kubeVipSecContext := dict }}
+            {{- if .Values.controlPlane.advanced.kubeVip.enabled }}
+            {{- $kubeVipSecContext = dict "capabilities" (dict "add" (list "NET_ADMIN" "NET_RAW")) }}
+            {{- end }}
+            {{- with merge (dict) $kubeVipSecContext .Values.controlPlane.statefulSet.security.containerSecurityContext }}
+              {{- if .capabilities }}
+                {{- if .capabilities.add }}
+                {{- $_ := set .capabilities "add" (uniq .capabilities.add) }}
+                {{- end }}
+              {{- end }}
+{{ toYaml . | indent 12 }}
+            {{- end }}
```

Comment:
Would be great if we add a simple helm unit test for this just to make sure this is set when kubeVip is enabled and not set when its disabled